### DISCUSSION TITLE
Most of the GDScript links are dead.

### DISCUSSION THREAD

**Discussion by kragil** [supabase/supabase#11889] [Questions]
In that Client Lib matrix only the first link for Godot works, the others are dead:

Godot Engine (GDScript) | supabase-gdscript | postgrest-gdscript | gotrue-gdscript | realtime-gdscript | storage-gdscript | functions-gdscript
-- | -- | -- | -- | -- | -- | --

[supabase-gdscript](https://github.com/supabase-community/godot-engine.supabase)	
[postgrest-gdscript](https://github.com/supabase-community/postgrest-gdscript)	
[gotrue-gdscript](https://github.com/supabase-community/gotrue-gdscript)	
[realtime-gdscript](https://github.com/supabase-community/realtime-gdscript)	
[storage-gdscript](https://github.com/supabase-community/storage-gdscript)	
[functions-gdscript](https://github.com/supabase-community/functions-gdscript)

Have the repos been deleted or moved?

**Comment by GaryAustin1** [comment]
You might need to ask as in issue in that repository.
But all the code seems to be in the  main repository.
https://github.com/supabase-community/godot-engine.supabase/tree/main/addons/supabase
![image](https://user-images.githubusercontent.com/54564956/214310605-dab5acc0-de96-4b16-a860-aa98467d5dca.png)
